spec/controllers/state_file/questions/nj_homeowner_property_tax_worksheet_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "rails_helper"
+
+RSpec.describe StateFile::Questions::NjHomeownerPropertyTaxWorksheetController do
+  let(:intake) { create :state_file_nj_intake }
+  before do
+    sign_in intake
+  end
+
+  describe "#show?" do
+    context "when indicated that they rent" do
+      let(:intake) { create :state_file_nj_intake, household_rent_own: "rent" }
+      it "does not show" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "when indicated neither rent nor own" do
+      let(:intake) { create :state_file_nj_intake, household_rent_own: "neither" }
+      it "does not show" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "when indicated both rent and own" do
+      let(:intake) { create :state_file_nj_intake, household_rent_own: "both" }
+      it "does not show" do
+        expect(described_class.show?(intake)).to eq false
+      end
+    end
+
+    context "when ineligible" do
+      let(:intake) { create :state_file_nj_intake, household_rent_own: "own", homeowner_home_subject_to_property_taxes: "no" }
```

Comment:
[pebble] would be nice to group these next several test contexts in a describe block for "when indicated own" starting here, so that it doesn't feel like the above 3 are missing the "own" case.